### ISSUE TITLE
Texture preview is not shown on Inspector

### ISSUE THREAD

**Issue by volzhs** [godotengine/godot#20476]
**Godot version:**

3.1.dev 832e2bfcd

**OS/device including version:**

Kubuntu 18.04

**Issue description:**

Texture preview is not shown on Inspector and gives error.
```
ERROR: generate: EditorResourcePreviewGenerator::generate needs to be overridden
   At: editor/editor_resource_preview.cpp:55.
```

**Steps to reproduce:**
1. create `Sprite`
2. assign texture
3. see no preview inspector and error on terminal console

**Minimal reproduction project:**

**Comment by guilhermefelipecgs** [comment]
Regression of #20137.

**Comment by volzhs** [comment]
I can see texture preview now. i guess it's fixed by #20573 in somehow.
